src/main/java/lotto/domain/Extractor.java

```diff
@@ -0,0 +1,35 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Extractor {
+    private static final String delimiter = " ";
```

Comment:
```suggestion
    private static final String DELIMITER = " ";
```

상수라면 대문자로 표현하는게 일반적일것 같아요 🤔

Reply:
넵 수정하겠습니다!